app/routes/apply/new-eligibility/about-you.js

```diff
@@ -12,30 +12,62 @@ const dateFormatter = require('../../../services/date-formatter')
 const benefitsHelper = require('../../../constants/benefits-enum')
 const NORTHERN_IRELAND = 'Northern Ireland'
 
+const REFERENCE_DOB_ERROR = '?error=expired'
+
 module.exports = function (router) {
   router.get('/apply/:claimType/new-eligibility/:dob/:relationship/:benefit/:referenceId', function (req, res) {
     UrlPathValidator(req.params)
 
+    if (!req.session ||
+        !req.session.claimType ||
+        !req.session.dobEncoded ||
+        !req.session.relationship ||
+        !req.session.benefit ||
+        !req.session.referenceId) {
+      return res.redirect(`/apply/first-time/new-eligibility${REFERENCE_DOB_ERROR}`)
+    }
+
+    var claimType = req.session.claimType
+    var dobEncoded = req.session.dobEncoded
+    var relationship = req.session.relationship
+    var benefit = req.session.benefit
+    var referenceId = req.session.referenceId
+
     return res.render('apply/new-eligibility/about-you', {
-      claimType: req.params.claimType,
-      dob: req.params.dob,
-      relationship: req.params.relationship,
-      benefit: req.params.benefit,
-      referenceId: req.params.referenceId
+      claimType: claimType,
+      dob: dobEncoded,
+      relationship: relationship,
+      benefit: benefit,
+      referenceId: referenceId
     })
   })
 
   router.post('/apply/:claimType/new-eligibility/:dob/:relationship/:benefit/:referenceId', function (req, res, next) {
     UrlPathValidator(req.params)
 
-    var dob = dateFormatter.decodeDate(req.params.dob)
-    var relationship = enumHelper.getKeyByAttribute(relationshipHelper, req.params.relationship, 'urlValue').value
-    var benefit = enumHelper.getKeyByAttribute(benefitsHelper, req.params.benefit, 'urlValue').value
-    var referenceAndEligibilityId = referenceIdHelper.extractReferenceId(req.params.referenceId)
+    if (!req.session ||
+        !req.session.claimType ||
+        !req.session.dobEncoded ||
+        !req.session.relationship ||
+        !req.session.benefit ||
+        !req.session.referenceId) {
+      return res.redirect(`/apply/first-time/new-eligibility${REFERENCE_DOB_ERROR}`)
+    }
+
+    var claimType = req.session.claimType
+    var dobEncoded = req.session.dobEncoded
+    var relationship = req.session.relationship
+    var benefit = req.session.benefit
+    var referenceId = req.session.referenceId
+
+    var dobFormatted = dateFormatter.decodeDate(dobEncoded)
+    var relationshipFormatted = enumHelper.getKeyByAttribute(relationshipHelper, relationship, 'urlValue').value
+    var benefitFormatted = enumHelper.getKeyByAttribute(benefitsHelper, benefit, 'urlValue').value
```

Comment:
relationship and benefit aren't formatted, it's just the key value. Only Dob variable needs to be supplied formatted